Runtime/AvatarCreator/Scripts/WebRequests/AvatarAPIRequests.cs

```diff
@@ -66,7 +63,7 @@ public async Task<List<AvatarTemplateData>> GetAvatarTemplates()
             var response = await authorizedRequest.SendRequest<Response>(
                 new RequestData
                 {
-                    Url = $"{RPM_AVATAR_V2_BASE_URL}/templates",
+                    Url = $"{RPM_AVATAR_V2_BASE_URL}/templates?{BODY_TYPE}={CoreSettingsHandler.CoreSettings.BodyType.GetDescription()}",
```

Comment:
is it possible that we might use this body type parameter somewhere else? 
If so we could just make a function on Core settings like 
CoreSettings.GetBodyTypeParameter 
that would return this 
$"bodyType={bodyType.GetDescription}"

Reply:
We are still storing the avatar bodyType within the core in some places. So I don't think that's a good way to go about it for now.

Reply:
okay makes sense lets leave as it is then.